bindings/python/test/test_datetime.py

```diff
@@ -98,38 +98,68 @@ def test_timezone_specified_in_schema(self):
         self.assertEqual(table, expected)
 
     def test_timezone_specified_in_codec_options(self):
-        # 1. When specified, CodecOptions.tzinfo will modify timestamp
-        #    type specifiers in the schema to inherit the specified timezone
-        tz = pytz.timezone("US/Pacific")
-        codec_options = CodecOptions(tz_aware=True, tzinfo=tz)
-        expected = Table.from_pydict(
-            {"_id": [1, 2], "data": self.expected_times},
-            ArrowSchema([("_id", int32()), ("data", timestamp("ms", tz=tz))]),
+        """Test behavior of setting tzinfo CodecOptions in Collection.with_options.
```

Comment:
What's the context for these datetime test changes?

Reply:
I expanded the test.  The original wasn't testing what it appeared to. Subleties between timestamps and datetimes.